mongodbatlas/resource_mongodbatlas_backup_compliance_policy.go

```diff
@@ -0,0 +1,516 @@
+	}
+	if v, ok := d.GetOk("policy_item_daily"); ok {
+		item := v.([]interface{})
+		itemObj := item[0].(map[string]interface{})
+		backupPolicyItem.FrequencyType = snapshotScheduleDaily
+		backupPolicyItem.RetentionUnit = itemObj["retention_unit"].(string)
+		backupPolicyItem.FrequencyInterval = itemObj["frequency_interval"].(int)
+		backupPolicyItem.RetentionValue = itemObj["retention_value"].(int)
+		backupPoliciesItem = append(backupPoliciesItem, backupPolicyItem)
+	}
+	if v, ok := d.GetOk("policy_item_weekly"); ok {
+		items := v.([]interface{})
+		for _, s := range items {
+			itemObj := s.(map[string]interface{})
+			backupPolicyItem.FrequencyType = snapshotScheduleWeekly
+			backupPolicyItem.RetentionUnit = itemObj["retention_unit"].(string)
+			backupPolicyItem.FrequencyInterval = itemObj["frequency_interval"].(int)
+			backupPolicyItem.RetentionValue = itemObj["retention_value"].(int)
+			backupPoliciesItem = append(backupPoliciesItem, backupPolicyItem)
+		}
+	}
+	if v, ok := d.GetOk("policy_item_monthly"); ok {
+		items := v.([]interface{})
+		for _, s := range items {
+			itemObj := s.(map[string]interface{})
+			backupPolicyItem.FrequencyType = snapshotScheduleMonthly
+			backupPolicyItem.RetentionUnit = itemObj["retention_unit"].(string)
+			backupPolicyItem.FrequencyInterval = itemObj["frequency_interval"].(int)
+			backupPolicyItem.RetentionValue = itemObj["retention_value"].(int)
+			backupPoliciesItem = append(backupPoliciesItem, backupPolicyItem)
+		}
+	}
+
+	backupPolicy.ScheduledPolicyItems = backupPoliciesItem
+	if len(backupPoliciesItem) > 0 {
+		backupCompliancePolicyUpdate.ScheduledPolicyItems = backupPoliciesItem
+	}
+
+	_, _, err := conn.BackupCompliancePolicy.Update(context.Background(), projectID, backupCompliancePolicyUpdate)
+	if err != nil {
+		return diag.FromErr(fmt.Errorf(errorBackupPolicyUpdate, projectID, err))
+	}
+
+	return resourceMongoDBAtlasBackupCompliancePolicyRead(ctx, d, meta)
+}
+
+func resourceMongoDBAtlasBackupCompliancePolicyDelete(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	// There is no resource to delete a backup compliance policy, it can only be updated.
+	log.Printf("[WARN] Note: Deleting Backup Compliance Policy resource in Terraform does not remove policy from Atlas Project. " +
+		"To disable a Backup Compliance Policy, the security or legal representative specified for the Backup Compliance Policy must contact " +
+		"MongoDB Support and complete an extensive verification process. ")
+
+	d.SetId("")
+	return nil
+}
+
+func resourceMongoDBAtlasBackupCompliancePolicyImportState(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	conn := meta.(*MongoDBClient).Atlas
+
+	parts := strings.SplitN(d.Id(), "-", 2)
+	if len(parts) != 1 {
+		return nil, errors.New("import format error: to import a Backup Compliance Policy use the format {project_id}")
```

Comment:
Minor suggestion for readability and capitalization consistency
```suggestion
		return nil, errors.New("Import format error: To import a Backup Compliance Policy, use the format {project_id}")
```

Reply:
Fixed

Reply:
Have to leave lowercase or go lint tests fail